tests/suites/host_test.function

```diff
@@ -432,6 +432,43 @@ static void write_outcome_result(FILE *outcome_file,
     fflush(outcome_file);
 }
 
+#if defined(__unix__) ||                                \
+    (defined(__APPLE__) && defined(__MACH__)) ||        \
+    defined(_WIN32)
+#define MBEDTLS_HAVE_CHDIR
+#endif
+
+#if defined(MBEDTLS_HAVE_CHDIR)
+/** Try chdir to the directory containing argv0.
+ *
+ * Failures are silent.
+ */
+static void try_chdir(const char *argv0) {
+    const char *slash = strrchr(argv0, '/');
```

Comment:
or you could just do

```
diff --git a/tests/suites/host_test.function b/tests/suites/host_test.function
index 94ae1a07e..c820f805b 100644
--- a/tests/suites/host_test.function
+++ b/tests/suites/host_test.function
@@ -444,7 +444,12 @@ static void write_outcome_result(FILE *outcome_file,
  * Failures are silent.
  */
 static void try_chdir(const char *argv0) {
-    const char *slash = strrchr(argv0, '/');
+    const char *slash = NULL;
+    for (const char *p = argv0; *p != '\0'; p++) {
+        if (*p == '/' || *p == '\\') {
+            slash = p;
+        }
+    }
     if (slash == NULL) {
         return;
     }
```

(yet, technically not the same, but the likelihood of a path on *nix having a `\` in it isn't something that a convenience assist should really worry about)